repair/row_level.cc

```diff
@@ -1417,9 +1413,16 @@ class repair_meta {
             return make_ready_future<>();
         }
         stats().rpc_call_nr++;
+        // Even though remote partitioner name is ignored in the current version of
+        // repair, we still have to send something to keep compatibility with nodes
+        // that run older versions. This will make it possible to run mixed cluster.
+        // Murmur3 is appropriate because that's the only supported partitioner at
+        // the time this change is introduced.
+        sstring remote_partitioner_name = "org.apache.cassandra.dht.Murmur3Partitioner";
```

Comment:
Why can't we send `_master_node_shard_config.partitioner_name` anymore?

Reply:
It is removed.